### PR TITLE
Datalab docs improvements

### PR DESCRIPTION
**Pull request by huiwengoh**
Mostly visual improvements to datalab docs:

- Added styling for the install md block to match other tutorials 
Before:
image
After:
image

- Fix some indentation issues
Before: 
image
After:
image

- Added link to function mentioned in tutorial

### CODE REVIEW

**Review by elisno — APPROVED** [review]

File: docs/source/tutorials/datalab/datalab_quickstart.ipynb [hunk 1]
```diff
@@ -700,12 +698,12 @@
...
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "This portion of the info shows overall label quality summaries of all examples annotated as a particular class (e.g. the `Label Issues` column is the estimated number of examples labeled as this class that should actually have a different label).\n",
-    "To learn more about this, see the documentation for the `cleanlab.dataset.rank_classes_by_label_quality` method.\n",
+    "To learn more about this, see the documentation for the [cleanlab.dataset.rank_classes_by_label_quality](../../cleanlab/dataset.html#cleanlab.dataset.rank_classes_by_label_quality)\n",
+    "method.\n",
```
Comment (jwmueller) [inline docs/source/tutorials/datalab/datalab_quickstart.ipynb:706]: this'll work across both stable/ and master/ branch versions of docs?

Comment (huiwengoh) [reply docs/source/tutorials/datalab/datalab_quickstart.ipynb:706]: It's a relative link so I think it should?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -304,7 +304,7 @@ Use relative linking to connect information between docs and jupyter notebooks,
     Here you have more control over the text that is displayed to display the module name.  When referring to a function that is alternatively defined in other modules as well, always use this option to be more explicit about which module you are referencing.
 - Link a tutorial (rst file) from within a source code docstring or rst file: ``:ref:`tutorial_name <tutorial_name>` ``
 - Link a tutorial notebook (ipynb file) from within a source code docstring or rst file: `` `notebook_name <tutorials/notebook_name.ipynb>`_ `` . (If the notebook is not the in the same folder as the source code, use a relative path)
-- Link a function from within a tutorial notebook: `[function_name](../cleanlab/file.rst#cleanlab.file.function_name)`
+- Link a function from within a tutorial notebook: `[function_name](../cleanlab/file.html#cleanlab.file.function_name)`
```
Comment (jwmueller) [inline DEVELOPMENT.md:307]: can you clarify this'll work across both stable/ and master/ branch versions of docs?

Comment (huiwengoh) [reply DEVELOPMENT.md:307]: Added a note!

Comment (elisno) [reply DEVELOPMENT.md:307]: I haven't verified it, but AIUI, the relative paths are relative to the configuration directory (the one with `conf.py`).


The relative links work across the branches.

**Review by jwmueller — APPROVED** [review]
Good stuff!

**Pull request merged**